### PR TITLE
Name change WinUI 3 Controls Gallery --> WinUI 3 Gallery 

### PR DESCRIPTION
**Pull request by gregwoo-microsoft**
Note: the project won't run until we rename the project file to WinUI3Gallery.DesktopWap.csproj

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: XamlControlsGallery/XamlControlsGallery.DesktopWap.Package.wapproj [hunk 1]
```diff
@@ -42,8 +42,8 @@
     <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
     <DefaultLanguage>en-US</DefaultLanguage>
     <AppxPackageSigningEnabled>True</AppxPackageSigningEnabled>
-    <EntryPointProjectUniqueName>XamlControlsGallery.DesktopWap.csproj</EntryPointProjectUniqueName>
-    <AppxBundleNameForOutput>XamlControlsGallery.DesktopWap</AppxBundleNameForOutput>
+    <EntryPointProjectUniqueName>WinUI3Gallery.DesktopWap.csproj</EntryPointProjectUniqueName>
```
Comment (gregwoo-microsoft) [inline XamlControlsGallery/XamlControlsGallery.DesktopWap.Package.wapproj:45]: Note: This change prevents us from building & running the solution until we rename the project file

File: XamlControlsGallery/app.manifest [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <assembly manifestVersion="1.0" xmlns="urn:schemas-microsoft-com:asm.v1">
-  <assemblyIdentity version="1.0.0.0" name="XamlControlsGallery.Desktop.app"/>
+  <assemblyIdentity version="1.0.0.0" name="WinUI3Gallery.Desktop.app"/>
```
Comment (gregwoo-microsoft) [inline XamlControlsGallery/app.manifest:3]: Same here

File: XamlControlsGallery/standalone.props [hunk 3]
```diff
@@ -9,15 +9,15 @@
     <ColorCodeVersion>2.0.13</ColorCodeVersion>
     <MicrosoftWindowsSDKBuildToolsNugetPackageVersion>10.0.22563-preview</MicrosoftWindowsSDKBuildToolsNugetPackageVersion>
     <MicrosoftCsWinRTPackageVersion>1.6.1</MicrosoftCsWinRTPackageVersion>
-    <!-- Where to place the files generated from CSWinRT from XamlControlsGallery.Desktop.-->
```
Comment (bpulliam) [inline XamlControlsGallery/standalone.props:12]: use WinUIGallery (drop the 3). This is main, so no version should be used.

File: XamlControlsGallery/app.manifest [hunk 4]
```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <assembly manifestVersion="1.0" xmlns="urn:schemas-microsoft-com:asm.v1">
-  <assemblyIdentity version="1.0.0.0" name="XamlControlsGallery.Desktop.app"/>
```
Comment (bpulliam) [inline XamlControlsGallery/app.manifest:3]: use WinUIGallery (drop the 3). This is main, so no version should be used.

File: XamlControlsGallery/XamlControlsGallery.DesktopWap.Package.wapproj [hunk 5]
```diff
@@ -42,8 +42,8 @@
     <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
     <DefaultLanguage>en-US</DefaultLanguage>
     <AppxPackageSigningEnabled>True</AppxPackageSigningEnabled>
-    <EntryPointProjectUniqueName>XamlControlsGallery.DesktopWap.csproj</EntryPointProjectUniqueName>
-    <AppxBundleNameForOutput>XamlControlsGallery.DesktopWap</AppxBundleNameForOutput>
```
Comment (bpulliam) [inline XamlControlsGallery/XamlControlsGallery.DesktopWap.Package.wapproj:46]: use WinUIGallery (drop the 3). This is main, so no version should be used.

File: XamlControlsGallery/XamlControlsGallery.DesktopWap.Package.wapproj [hunk 6]
```diff
@@ -42,8 +42,8 @@
     <TargetPlatformMinVersion>10.0.17763.0</TargetPlatformMinVersion>
     <DefaultLanguage>en-US</DefaultLanguage>
     <AppxPackageSigningEnabled>True</AppxPackageSigningEnabled>
-    <EntryPointProjectUniqueName>XamlControlsGallery.DesktopWap.csproj</EntryPointProjectUniqueName>
```
Comment (bpulliam) [inline XamlControlsGallery/XamlControlsGallery.DesktopWap.Package.wapproj:45]: use WinUIGallery (drop the 3). This is main, so no version should be used.

File: XamlControlsGallery/XamlControlsGallery.DesktopWap.Package.wapproj [hunk 7]
```diff
@@ -82,7 +82,7 @@
     </AppxManifest>
   </ItemGroup>
   <ItemGroup>
-    <ProjectReference Include="XamlControlsGallery.DesktopWap.csproj">
```
Comment (bpulliam) [inline XamlControlsGallery/XamlControlsGallery.DesktopWap.Package.wapproj:85]: use WinUIGallery (drop the 3). This is main, so no version should be used.

File: XamlControlsGallery/SettingsPage.xaml [hunk 8]
```diff
@@ -116,19 +116,19 @@
...
                     </Paragraph>
                     <Paragraph>
                         If you would like a new sample, request it on GitHub:
                     </Paragraph>
                     <Paragraph Margin="0,10,0,10">
-                        <Hyperlink NavigateUri="https://github.com/microsoft/Xaml-Controls-Gallery/issues/new?template=sample_request.md">New Sample Request</Hyperlink>
+                        <Hyperlink NavigateUri="https://github.com/microsoft/WinUI-Gallery/issues/new?template=sample_request.md">New Sample Request</Hyperlink>
                     </Paragraph>
                     <Paragraph>
-                        If you found a bug in your app (not in the Xaml Controls Gallery) and need help, file an issue on the WinUI GitHub:
```
Comment (bpulliam) [inline XamlControlsGallery/SettingsPage.xaml:131]: use WinUIGallery (drop the 3). This is main, so no version should be used.

File: XamlControlsGallery/SettingsPage.xaml [hunk 9]
```diff
@@ -116,19 +116,19 @@
                     Text="Found a bug? Want a new sample?" />
                 <RichTextBlock MaxWidth="840" Margin="0,10,0,0" IsTextSelectionEnabled="True">
                     <Paragraph>
-                        If you have found a bug within the Xaml Controls Gallery, please file an issue on GitHub:
+                        If you have found a bug within the WinUI 3 Gallery, please file an issue on GitHub:
                     </Paragraph>
                     <Paragraph Margin="0,10,0,10">
-                        <Hyperlink NavigateUri="https://github.com/microsoft/Xaml-Controls-Gallery/issues/new?template=bug_report.md">New Xaml Controls Gallery Bug Report</Hyperlink>
```
Comment (bpulliam) [inline XamlControlsGallery/SettingsPage.xaml:122]: This is a good place for a fork - separate bug templates for 2 and main.

**Review by bpulliam — CHANGES_REQUESTED** [review]

**Comment by gregwoo-microsoft** [comment]
@bpulliam, just made the changes you suggested

**Review by bpulliam — APPROVED** [review]

**Pull request merged**